### ISSUE TITLE
why you use layer_norm ? 

### ISSUE THREAD

**Issue by taki0112** [NVlabs/MUNIT#10]
Hi
[code](https://github.com/NVlabs/MUNIT/blob/master/networks.py#L198)
In Decoder,  you used the `layer_norm`

have you tried another normalization ? like instance, batch, group

**Comment by xunhuang1995** [comment]
Instance norm does not work well, since it removes the global feature mean and variance that capture the style information.

Batch norm is the same since we use batch size = 1.

The paper had been submitted before group normalization came out. So we haven't try it.

**Comment by taki0112** [comment]
I have some question...

If you look at configs, there is no information about animal translation.
When will you upload it?
Is it ok to just run as hyper-parameter like edge2shoes?
